_pages/business-units/18f/projects-partners/how-we-collaborate.md

```diff
@@ -18,7 +18,7 @@ Each time a new team forms—or anytime someone joins an existing team—is a ch
 
 - **Use introductions to set the tone.** Take a moment at the beginning of each meeting to make sure everyone’s met, and (if not) introduce the team and welcome new attendees. In introductions, include your name, pronouns, and role. Location can also be nice, if only to know what time it is for everyone.
 - **Get to know each other!** If you’re working with someone new, set up a virtual coffee or tea to learn about their life, background, and strengths. A little investment in the relationship often pays off later as you collaborate.
-- **Establish team norms** for communication, working hours, feedback, and decision-making. A [team charter](https://docs.google.com/document/d/1J8MqOA-JkLKzSFTVxeXDQub7ZTsWuNq5nOZ_ACICdig/edit#):lock: can help facilitate this conversation.
```

Comment:
The lock icon is added by private eye and `:lock:` doesn't render into anything in the HTML, so we can remove this.